examples/tests/test_daal4py_examples.py

```diff
@@ -0,0 +1,400 @@
+# ==============================================================================
```

Comment:
I would suggest not to use examples/tests at all, to keep examples folder clean and tidy for users. 

If we consider this as tests, then lets keep in tests folder

Reply:
I was faster than your comment :) It's back in the "old" `tests` folder.